src/Components/Aspire.Azure.Messaging.ServiceBus/AzureMessagingServiceBusSettings.cs

```diff
@@ -87,11 +88,35 @@ void IConnectionStringSettings.ParseConnectionString(string? connectionString)
             if (!connectionString.Contains(';'))
             {
                 FullyQualifiedNamespace = connectionString;
+                return;
             }
-            else
+
+            var connectionBuilder = new DbConnectionStringBuilder()
+            {
+                ConnectionString = connectionString
+            };
+
+            // Note: Strip out the EntityPath from the connection string in order
+            // to tell if we are left with just Endpoint.
+            // The EntityPath can contain a queue or topic name. And if it references a topic,
+            // it can contain {topic}/Subscriptions/{subscription}. See https://github.com/Azure/azure-sdk-for-net/pull/27070
+
+            if (connectionBuilder.TryGetValue("EntityPath", out var _))
```

Comment:
Can be removed, equivalent to `connectionBuilder.Remove("EntityPath");` which returns `bool`

Reply:
In the future we will use the EntityPath value to initialize the queue/topic name, and optionally the subscription name.